src/ApiService/Tests/OrmModelsTest.cs

```diff
@@ -0,0 +1,614 @@
+using FsCheck;
+using FsCheck.Xunit;
+using Xunit.Abstractions;
+using Microsoft.OneFuzz.Service;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+using System.Collections.Generic;
+using System;
+using System.Linq;
+using System.Security;
+using System.Text.Json;
+
+namespace Tests
+{
+
+    public class OrmGenerators
+    {
+
+        public static Gen<BaseEvent> BaseEvent()
+        {
+            return Gen.OneOf(new[] {
+                Arb.Generate<EventNodeHeartbeat>().Select(e => e as BaseEvent),
+                Arb.Generate<EventTaskHeartbeat>().Select(e => e as BaseEvent),
+                Arb.Generate<EventInstanceConfigUpdated>().Select(e => e as BaseEvent)
+            });
+        }
+
+
+        public static Gen<Uri> Uri()
+        {
+            return Arb.Generate<IPv4Address>().Select(
+                arg => new Uri($"https://{arg.Item.ToString()}:8080")
+            );
+        }
+
+        public static Gen<WebhookMessageLog> WebhookMessageLog()
```

Comment:
can you only generate then `Event` an `EventType` fields these based on BaseEvent generator.
This is currently generating event that are no implemented yet

Reply:
I added it anyways